src/core/ucc_lib.c

```diff
@@ -213,16 +213,13 @@ static ucc_status_t ucc_cl_lib_init(const ucc_lib_params_t *user_params,
     return status;
 }
 
-static int ucc_tl_is_required(ucc_lib_info_t *lib, ucc_tl_iface_t *tl_iface,
-                              int explicitly)
+static int ucc_tl_is_required(ucc_lib_info_t *lib, ucc_tl_iface_t *tl_iface)
```

Comment:
is_required and is_requested almost identical, maybe merge them

Reply:
done